server/src/test/java/org/opensearch/cluster/routing/MovePrimaryFirstTests.java

```diff
@@ -60,7 +56,8 @@ protected void createAndIndex(String index, int replicaCount, int shardCount) {
     public void testClusterGreenAfterPartialRelocation() throws InterruptedException {
         internalCluster().startMasterOnlyNodes(1);
         final String z1 = "zone-1", z2 = "zone-2";
-        final int primaryShardCount = 100;
+        // Primary shard count should be even for equal distribution across two nodes
```

Comment:
You could assert this in code to avoid someone changing it accidentally.

Reply:
Something like the following then you don't need the comment:

```
assertTrue("Primary shard count must be even for equal distribution across two nodes", primaryShardCount % 2 == 0);
```

Reply:
Makes sense, done!